.travis.yml

```diff
@@ -14,8 +14,25 @@ matrix:
     - os: osx
       compiler: clang
 
-script:
-  - make coverage
+- stage: coverage
+  script:
+    - make coverage
+
+- stage: cmake_project
+  script:
+  # install header only lib with cmake
+  - rm -rf build || true
+  - mkdir build
+  - cd build
+  - cmake ..
+  - sudo cmake --build . --target install
+  # build example project using cmake find_package
+  - cd ../example
+  - mkdir example_build
+  - cd example_build
+  - cmake ..
+  - make
+  - my_executable
```

Comment:
I'm dont really know how travis/appveyor works, I'm a gitlab ci guy...

though this would be kind of the test to see if find_package works